hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystemWithFSO.java

```diff
@@ -286,4 +292,35 @@ public void testLeaseRecoverable() throws Exception {
     assertTrue(fs.recoverLease(source));
     assertTrue(fs.isFileClosed(source));
   }
+
+  @Test
```

Comment:
This is fine. But you can also make the same unit test in TestOMKeySetTimesRequestWithFSO.java which is more lightweight (doesn't need to deal the entire cluster, client interactions just need to test OMKeySetTimesRequestWithFSO )

Reply:
done